mlflow/tensorflow.py

```diff
@@ -651,8 +651,8 @@ class __MLflowTfKerasCallback(Callback):
         Records model structural information as params after training finishes.
         """
 
-        def __init__(self):
-            pass
```

Comment:
@dbczumar I checked and this behaves the same way if super() was not called. So no need to invoke parent class's constructor. 
<img width="361" alt="Screen Shot 2020-11-12 at 1 56 58 PM" src="https://user-images.githubusercontent.com/73549313/99009283-a4318780-24fc-11eb-8dec-17a10a4656e2.png">

Reply:
Awesome :)